### PR TITLE
Add QR code scanner fragment (EXPOSUREAPP-8952)

### PR DESCRIPTION
**Pull request by SamuraiKek**
From the organizer warn flow, scan an event qr code -> `TraceLocationWarnDurationFragment` is shown after scanning qr and event info card is updated correctly.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_warn_qr_scanner_fragment.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/trace_location_organizer_warn_qr_scanner_fragment.xml:1]: can't we use the same layout?

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/layout/trace_location_organizer_warn_qr_scanner_fragment.xml:1]: Ah, you're right. Let me change it.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/qrcode/OrganizerWarnQrCodeScannerViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+) : CWAViewModel() {
+    val events = SingleLiveEvent<OrganizerWarnQrCodeNavigation>()
+
+    fun onNavigateUp() {
+        events.postValue(OrganizerWarnQrCodeNavigation.BackNavigation)
+    }
+
+    fun onScanResult(barcodeResult: BarcodeResult) = launch {
+        try {
+            Timber.i("uri: $barcodeResult.result.text")
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/warn/qrcode/OrganizerWarnQrCodeScannerViewModel.kt:29]: "${barcodeResult.result.text}"

File: Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml [hunk 3]
```diff
@@ -59,7 +59,7 @@
         android:id="@+id/scanCheckInQrCodeFragment"
         android:name="de.rki.coronawarnapp.ui.presencetracing.attendee.scan.ScanCheckInQrCodeFragment"
         android:label="ScanCheckInQrCodeFragment"
-        tools:layout="@layout/fragment_scan_qr_code" />
+        tools:layout="@layout/fragment_scan_qr_code"></fragment>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml:62]: why ?

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml:62]: Had added nav args before duplicating the Qr fragment and forgot to delete this.

**Review by mtwalli — APPROVED** [review]
lgtm

**Review by BMItr — APPROVED** [review]

**Pull request merged**